noxfile.py

```diff
@@ -203,9 +203,8 @@ def build_release(session):
     session.log("# Checkout the tag")
     session.run("git", "checkout", version, external=True, silent=True)
 
-    session.log("# Cleanup build/ before building the wheel")
-    if release.have_files_in_folder("build"):
-        shutil.rmtree("build")
+    session.log("# Wipe Git-untracked files before building the wheel")
+    session.run("git", "clean", "-fxd", external=True, silent=True)
```

Comment:
See my comment, which should have been submitted as a review instead 🙁

Reply:
(already answered: https://github.com/pypa/pip/pull/7631#issuecomment-576755948)